### DISCUSSION TITLE
Make the textarea more beautiful

### DISCUSSION THREAD

**Discussion by wordpressvn** [jeffreyvr/wp-settings#12] [Ideas]
I propose adding rows="10" to the file vendor\jeffreyvanrossum\wp-settings\resources\views\options\textarea.php

**Comment by jeffreyvr** [comment]
I'm thinking of making it on option to set so people can choose themselves how many rows they would like to apply.

Will probably be as simple as:

```php
$section->add_option('textarea', [
    'name' => 'option_1',
    'label' => __('Option 1', 'textdomain'),
    'rows' => 10
]);
```
